test/fixtures/state-saved-v1-7.js

```diff
@@ -751,6 +751,9 @@ export const savedStateV1InteractionCoordinate = {
           },
           coordinate: {
             enabled: true
+          },
+          geocoder: {
```

Comment:
the saved state should not be changed, the merged config should have geocoder set to `{enablced: false}`